index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8" />
+    <title>TypingPractice</title>
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="game-box" id="game">
+      <input type="text" class="input-field" />
+      <div class="typing-practice-game-container">
+        <input
+          type="number"
+          placeholder="Generated Words"
+          class="word-count"
+          id="word-count"
+          min="5"
+          max="40"
+          onKeyDown="return false"
+        />
+        <button onclick="generateWords()">Generate</button>
+        <div class="typing-text">
+          <p id="text-to-write"></p>
+        </div>
+        <div class="credits-retry">
+          <p>TypingPractice Game by Dustin Koch</p>
+          <button>Try Again</button>
+        </div>
+      </div>
+    </div>
+
+    <div class="scoreboard" id="scoreboard">
+      <div class="score-container">
+        <p>Time is up!</p>
+        <br />
+        <p>Your Total Best</p>
+        <div class="overall-best-score">
+          <div class="left">Mistakes: <span id="alltime_mistakes">0</span></div>
```

Comment:
Classes like "left" and "right" are not good class names.
Stick to BEM Notation here as well.
```suggestion
          <div class="alltime-mistakes">Mistakes: <span id="alltime_mistakes">0</span></div>
```

Reply:
Those two class names where from testing, they aren't needed anymore. I removed them